CommandLine/libEGM/tmx.h

```diff
@@ -50,13 +45,35 @@ struct HexMapUtil {
   std::string staggerIndex;
 };
 
+
+/**
+ * @brief The HexMapUtil struct captures details of a hexagonal Tiled map
+ */
+struct HexMapUtil {
+  /**
+   * @brief Length of side of hexagonal tile
+   */
+  unsigned int hexSideLength;
+
+  /**
+   * @brief Hex map is a staggered map
+   */
+  StaggerUtil staggerUtil;
```

Comment:
There may be reasons to prefer aggregation, but in this case I would be inclined to derive `HexMapUtil` from `StaggerUtil` instead.

Regarding the naming, I'm a bit confused about the choice of "Util", since at least personally I expect a utility to do something rather than just store some data. Maybe "Info", "Parameters" or "Properties" would be more suitable (or shorter versions, if you prefer).

Reply:
Admittedly I'm pretty bad at naming, updated those structs with Info suffix and inherited StaggerInfo from HexMapInfo